docs/read-the-docs/source/workceptor.rst

```diff
@@ -0,0 +1,182 @@
+Workceptor
+==========
+
+Workceptor is a component of receptor that handles units of work.
+
+``work-command`` defines a type of work that can run on the node.
+
+foo.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: foo
+
+    - log-level:
+        level: Debug
+
+    - tcp-listener:
+        port: 2222
+
+    - control-service:
+        service: control
+        filename: /tmp/foo.sock
+
+    - work-command:
+        workType: echosleep
+        command: bash
+        params:  "-c \"for i in {1..5}; do echo $i; done\""
+
+bar.yml
+
+.. code-block:: yaml
+
+    ---
+    - node:
+        id: bar
+
+    - log-level:
+        level: Debug
+
+    - tcp-peer:
+        address: localhost:2222
+
+    - control-service:
+        service: control
+
+    - work-command:
+        worktype: echosleep
+        command: bash
+        params:  "-c \"for i in {1..10}; do echo $i; sleep 1; done\""
+
+
+Configuring work commands
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``worktype`` User-defined name to give this work definition
+
+``command`` The executable that is invoked when running this work
+
+``params`` Command-line options passed to this executable
+
+
+Local work
+^^^^^^^^^^
+
+Start the work by connecting to the ``control-service`` and issuing a "work submit" command
+
+.. code-block::
+
+    $ receptorctl --socket /tmp/foo.sock work submit echosleep --no-payload
```

Comment:
`work submit` command is missing target node argument (this was listed as required in the `interacting_with_nodes.rst` section)

Reply:
yes, receptorctl does a lot of convenient things like defaulting to the local node if --node is not provided.

I think I need to make it more clear in that section in interacting_with_nodes that the required/optional parameters are for when connecting to the socket directly. Under the hood, receptorctl will be following the requirements on that table, but the user won't be aware of it.